src/Sylius/Bundle/ApiBundle/Resources/config/api_resources/Adjustment.xml

```diff
@@ -0,0 +1,52 @@
+<?xml version="1.0" ?>
+
+<!--
+
+ This file is part of the Sylius package.
+
+ (c) Paweł Jędrzejewski
+
+ For the full copyright and license information, please view the LICENSE
+ file that was distributed with this source code.
+
+-->
+
+<resources xmlns="https://api-platform.com/schema/metadata"
+           xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+           xsi:schemaLocation="https://api-platform.com/schema/metadata https://api-platform.com/schema/metadata/metadata-2.0.xsd"
+>
+    <resource class="%sylius.model.adjustment.class%" shortName="Adjustment">
+        <attribute name="normalization_context">
+            <attribute name="groups">
+                <attribute>adjustment:read</attribute>
+            </attribute>
+        </attribute>
+
+        <collectionOperations />
+
+        <itemOperations>
+            <itemOperation name="get">
+                <attribute name="security">is_granted('ROLE_API_ACCESS')</attribute>
+            </itemOperation>
+        </itemOperations>
+
+        <subresourceOperations>
+            <subresourceOperation name="api_order_items_adjustments_get_subresource">
```

Comment:
What is the difference between these two operations?

Reply:
Using the subresources to generate adjustments endpoints, for example to have endpoint: `/orders/{token}/items/{id}/adjustments`, we need to expose also ``/order-items/{id}/adjustments``. 

These are all adjustments endpoints that are currently exposed:
![image](https://user-images.githubusercontent.com/6140884/90729728-4c053380-e2c7-11ea-8a47-3fc23e9023d0.png)